### ISSUE TITLE
Thank You! I didn’t know how else to say it here

### ISSUE THREAD

**Issue by ArtOfZenDigital** [FCare/Kronos#1260]
What's the full name of the game? Zero Divide: The Final Conflict (Japan)

What's your frontend ? Stand Alone Windows 10 Kronos version 2.2.1 64 Bit

What's your OS? Windows 10 Pro 64bit

What's your hardware? Intel i9 9900K at 5ghz, Nvidia RTX 3080 TI, Adata XPG Pro 1Tb M2 SSD, MSI MPG Z390 Gaming Plus Motherboard

What's the storage type of the game? CD image created from my original Sega Saturn game CD

What's the format of your CD image? cue/bin

I just want to say thank you for addressing the VDP2 floor scaling issue in Zero Divide. I’m not on Discord, and there’s no other way that’s possible to say thanks here. My previous issue was the way the VDP2 floor surfaces were scaling in Zero Divide. The latest official Kronos release fixes the issue perfectly, bringing up the overall quality of the image. Fcare, your hard work is greatly appreciated, and it’s the reason why my Saturn isn’t hooked up to my TV right now, you even have the Saturn CD player and screen saver fully emulated properly. My Sega history video can be completed now since every game will be represented with the best presentation possible. Best wishes!

The first screen shot is how Zero Divide looked in Kronos 2.1.5 in 3840x2160. (The screen shot is rendered at 2560x1440)

![Zero Divide SS 4K No RBG mp4_snapshot_00 32 874](https://user-images.githubusercontent.com/91174966/138126321-a38863ac-0c81-4f4e-9f1b-53962847b5d6.jpg)

The second screen shot is how Zero Divide looks in Kronos 2.2.0 in 3840x2160. (The screen shot is rendered at 2560x1440) The 2 screen shots may be at different time stamps, but the improvements in VDP2 scaling are drastic.

![kronos 10-20-2021 10-13-34 AM mp4_snapshot_00 18 624](https://user-images.githubusercontent.com/91174966/138126351-7ee7a56b-c7fa-4694-8f33-69bb5ee0be3c.jpg)

**Comment by BenjaminSiskoo** [comment]
Thanks a lot :)

**Comment by ArtOfZenDigital** [comment]
> Thanks a lot :)

FCare mentioned that the scaling fixes in the latest build were a small increase in VDP2 quality, but it’s definitely a lot better than that all around. The advancements definitely shouldn’t be downplayed. The floor surfaces in Zero Divide look razor sharp now, and the texture mapping is very sharp looking as well. Out of curiosity, are the new settings such as de-interlacing bob, etc, specific to certain Saturn games? I have a lot of games for my Saturn, but most of them were import fighting games like Pocket Fighter, Dead or Alive, Marvel Vs Street Fighter, Street Fighter Zero 3, Zero Divide. As far as RPGs go, Grandia and Panzer Dragoon Saga. I was lucky enough to preorder that game along with Burning Rangers. All of the disc games I’ve transferred to my PC to play on Kronos work perfectly without the new settings.

As far as Kronos, I know that some games aren’t compatible, but they’re truly few and far between. Nearly every important, or even great, but obscure Saturn game is playable with massive visual improvements. Once Zero Divide was playable, for me, it was amazing, because it really showed what the Saturn could do in terms of a high res game with Gouraud Shading, something many people, Sega included, thought was impossible. The high res support for the VDP2 in the latest official build is incredible, especially after the initial shock of seeing Zero Divide added to the Kronos compatibility list last December. It’s such a little known 3D fighting gem that very few people knew about when it came out.

The Saturn was easily my favorite system during the 32 bit era, and when my sister got married, and her husband gave me his PS1, I only had a small handful of games for it. The Saturn was still my go to system. I can’t thank you guys enough, because the official RetroBit Saturn pad works perfectly with Kronos, with  simply stunning graphics as well. Thank you again for pulling off a miracle with Saturn emulation, from the unbelievably beautiful RBG VDP2 scaling, to the top notch performance, allowing my older HTPC with an Intel i5 2500 & 1050Ti to use as my second Saturn in a smooth 1080p 60. Mednafen isn’t user friendly and despite its high compatibility, playing my actual Saturn through an RGB to HDMI cable looks better than Mednafen. Kronos, on the other hand, makes the games I love look the way my nostalgic memory remembers seeing them. You, and everyone involved in this project have breathed new life into Yabuse, and turned it into the best performing, best looking, and most compatible Saturn emulators that offers both VDP1 and VDP2 upscaling. You don’t owe anyone anything, but you have provided a lot nostalgia for a great deal of Sega fans.

**Comment by FCare** [comment]
Hi, thanks for your support. It is always nice to read that kronos is appreciated :-)
Regarding new settings, in old versions of kronos, in case of interlaced screen, a simple and inaccurate solution was implemented to de-interlace the image such that it looks not so bad on LCD screen. But we found that it was reducing vertical resolution for most of games (like dead or Alive or Last Bronx). So we decided to display the game as interlaced and, depending user wish or screen, to allow different de-interlacing method such as BOB, OSSC clone or by simulating a CRT scanline.
As a result, we are trying to deinterlace only when there is effectively an interlaced screen and if the user has chosen a possibility.
If you play on a CRT screen, in full screen, I will assume it will be almost useless. But on LCD, it will fixes the interlacing and still allow to get an effect of improved vertical resolution.
